source/features/show-user-top-repositories.tsx

```diff
@@ -23,5 +23,8 @@ void features.add(import.meta.url, {
 	include: [
 		pageDetect.isUserProfileMainTab,
 	],
+	exclude: [
+		pageDetect.isGist,
+	],
```

Comment:
It's ok here, but it feels like we should treat `isGist` similarly to what we do with is404: _If the includes/asLongAs have isGist, then enable the feature there, if not don't._ There might only be a handful of situation where features can still apply to isGist, like hasCode, but maybe it's better to edit those instead, e.g.

```js
include: [
	hasCode,
	isGist,
]